### PR TITLE
Add grab cursor for ie

### LINKED ISSUE #1139: No grab cursor on IE
The grab (and grabbing) cursor are not supported on IE, we should add a fallback: use `cursor: move;` or a custom image `cursor: url(grab.cur);`.

### PR DESCRIPTION
**Pull request by sbrunner**
Don't succeed to test it @ochriste can you test it?

demo: https://camptocamp.github.io/ngeo/grab-cursor/examples/contribs/gmf/apps/desktop_alt/?lang=fr&baselayer_ref=map

Should fix #1139

### CODE REVIEW

File: contribs/gmf/less/desktoplayertree.less [hunk 1]
```diff
@@ -81,6 +82,7 @@ gmf-layertree {
   }
 
   .sortable-handle {
+    cursor: url(../cursors/grab.cur);
```
Comment (fredj) [inline contribs/gmf/less/desktoplayertree.less]: the fallback is missing, must be: `cursor: url(../cursors/grab.cur), default;`

File: contribs/gmf/less/desktoplayertree.less [hunk 2]
```diff
@@ -35,8 +35,9 @@ gmf-layertree {
 
   &.dragger {
     * {
-      cursor: grabbing;
+      cursor: url(../cursors/grabbing.cur);
```
Comment (fredj) [inline contribs/gmf/less/desktoplayertree.less]: the fallback is missing, must be: `cursor: url(../cursors/grabbing.cur), default;`

**Comment by sbrunner** [comment]
@fredj done :-)

**Comment by fredj** [comment]
LGTM

**Pull request merged**